src/index.html

```diff
@@ -17,11 +17,377 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="style.scss"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
     />
   </head>
 
+
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="/src/images/logo.png"
```

Comment:
The path to your image file is absolute. This can cause problems if the file structure of your project changes. It's better to use relative paths.